include/anvill/Specification.h

```diff
@@ -124,6 +124,12 @@ class Specification {
   // Return the architecture used by this specification.
   std::shared_ptr<const remill::Arch> Arch(void) const;
 
+  // Return the image name used by this specification.
+  std::string ImageName(void) const;
```

Comment:
Should return a const ref here to avoid copying on every call.

Reply:
[ccbac6a](https://github.com/lifting-bits/anvill/pull/379/commits/ccbac6a4749f767dabadc244e8739256f5c5e46a)